safe/common/parameters/default_select_parameter_widget.py

```diff
@@ -151,3 +152,29 @@ def toggle_custom_value(self):
             self.custom_value.setDisabled(False)
         else:
             self.custom_value.setDisabled(True)
+
+    def toggle_input(self):
+        """Change behaviour of radio button based on input."""
+        # If input is a field, then disable all radio button.
+        if self.input.currentText() != 'No Field':
```

Comment:
I don't know how @ismailsunni build the QComboxBox. If there is some data in the comboxbox, I would really prefer this way.
But as a quick fox for now, if you don't have `data`, ok to use this definition.